Makefile

```diff
@@ -5,12 +5,13 @@ GODEP_PATH=$(shell pwd)/Godeps/_workspace
 ORIGINAL_PATH=$(shell echo $(GOPATH))
 COMBINED_GOPATH=$(GODEP_PATH):$(ORIGINAL_PATH)
 
-.PHONY: packages deb test linux darwin
+.PHONY: packages deb test linux darwin windows
 
-all: deb linux darwin docker
+all: deb linux darwin docker windows
```

Comment:
I would put `windows` before docker here, just so it builds all 3 binaries together.